Servers/UI/OJS.Servers.Ui/ClientApp/src/common/url-types.ts

```diff
@@ -84,17 +84,28 @@ interface IContestEditUrlParams {
     id: number;
 }
 
+interface ITerm {
+    key: string;
+    value: string;
+}
+
 interface IGetSearchResultsUrlParams {
+    searchTerm: string;
+    selectedTerms: ITerm[];
+}
+
+interface IGetSearchResultsParams {
     searchTerm: string;
     page: number;
-    selectedTerms: [];
+    searchCategory: string;
```

Comment:
Use int and map to enum in BE

Reply:
After discussing this part was decided to be transferred to separate issue

Reply:
Please log an issue